dotnet/src/Connectors/Connectors.Ollama.UnitTests/Services/OllamaChatCompletionTests.cs

```diff
@@ -0,0 +1,209 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.Ollama;
+using Microsoft.SemanticKernel.Connectors.Ollama.Core;
+using Xunit;
+
+namespace SemanticKernel.Connectors.Ollama.UnitTests;
+
+public class OllamaChatCompletionTests : IDisposable
```

Comment:
Make this class more compact. Blank lines between Arrange/act/assert is sufficient. Remove blank lines between single statements.